### PR TITLE
release: 2024-11-04

### PR DESCRIPTION
**Pull request by sentientwaffle**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,60 @@
...
+## TigerBeetle 0.16.12
+
+Released: 2024-11-04
+
+### Safety And Performance
+
+- [#2356](https://github.com/tigerbeetle/tigerbeetle/pull/2356)
+
+  Add "Vortex" – a full-system integration test.
+  Notably, unlike to VOPR this test suite covers the language clients.
```
Comment (matklad) [inline CHANGELOG.md]:
```suggestion
  Notably, unlike VOPR this test suite covers the language clients.
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -3,6 +3,60 @@
...
+  Fix multiversion builds on MacOS.
+
+- [#2442](https://github.com/tigerbeetle/tigerbeetle/pull/2442)
+
+  On an unrecognized error code from the OS, print that error before we panic.
+  (This was already the policy in `Debug` builds, but now it includes `ReleaseSafe` as well.)
+
+- [#2444](https://github.com/tigerbeetle/tigerbeetle/pull/2444)
+
+  Fix a panic involving an in-flight w rite to an old reply after state sync.
```
Comment (matklad) [inline CHANGELOG.md]: A lot of safety in this release!

File: CHANGELOG.md [hunk 3]
```diff
@@ -3,6 +3,60 @@
...
+### Internals
+
+- [#2440](https://github.com/tigerbeetle/tigerbeetle/pull/2440)
+
+  Expose reply timestamp from `vsr.Client`.
+  (Note that this is not yet surfaced by language clients).
+
+### TigerTracks 🎧
+
+- [Everything In Its Right Place](https://www.youtube.com/watch?v=onRk0sjSgFU)
```
Comment (matklad) [inline CHANGELOG.md:58]: ⚡

**Review by matklad — APPROVED** [review]

**Comment by sentientwaffle** [comment]
No release this week due to a flaky integration test failure (https://github.com/tigerbeetle/tigerbeetle/pull/2453)

**Pull request closed without merging**